project/settings.py

```diff
@@ -126,7 +126,7 @@
 
 ROOT_URLCONF = "project.urls"
 LOGIN_URL = "/login/"
-EXPERIMENTER_LOGIN_URL = "/accounts/login/"
+RESEARCHER_LOGIN_URL = "/accounts/login/"
```

Comment:
I think we can leave this since referncing the app - it also makes me a little nervous that we don't change this somewhere else :)